etc/plugin/go-build.sh

```
@@ -1,12 +1,14 @@
 #!/bin/bash
+set -e
+BUILD_FLAGS="-trimpath -ldflags='-s -w'"
```

Shellcheck (0.11.0)

[warning] Quotes/backslashes will be treated literally. Use an array.

(SC2089)